source/index.html.md

```diff
@@ -992,6 +992,25 @@ equityStages | List of int | TODO
 exitStages | List of int | TODO
 ids | List of int | TODO
 
+## Investors Filter
+
+| Parameter name              | Parameter type | Description                                                                       |
+|-----------------------------|----------------|-----------------------------------------------------------------------------------|
+| investorTypeIDs             | List of int    | See Section [Investor Types](#investor-types) for the accepted values             |
+| includeOtherInvestorTypes   | boolean        | TODO                                                                              |
```

Comment:
This field is quite confusing since in the Investor types we have 'Other', and on the platform UI we have 'include other investor types' which is not related to the mentioned 'Other' type and returns completely different results

Reply:
yes, this filter is for adding investors with a secondary type matching one of the selected types, I'll replace the TODOs after the merge